values.yaml

```
@@ -286,39 +342,98 @@ server:
       memory: "100Mi"
       cpu: "100m"
 
-  # updatePartition is used to control a careful rolling update of Consul
-  # servers. This should be done particularly when changing the version
-  # of Consul. Please refer to the documentation for more information.
+  # This value is used to carefully
+  # control a rolling update of Consul server agents. This value specifies the
+  # partition (https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions)
+  # for performing a rolling update. Please read the linked Kubernetes documentation
+  # for more information.
```

Could we link https://www.consul.io/docs/k8s/upgrade#upgrading-consul-servers here as well?
```suggestion
  # This value is used to carefully
  # control a rolling update of Consul server agents. This value specifies the
  # partition (https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions)
  # for performing a rolling update. Please read the linked Kubernetes documentation
  # and https://www.consul.io/docs/k8s/upgrade#upgrading-consul-servers for more information.
```

---

👍